lib/pharos/etcd/client.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+require 'json'
+
+module Pharos
+  module Etcd
+    class Client
+      CURL = 'sudo curl -s --connect-timeout 2 --cacert /etc/pharos/pki/ca.pem --cert /etc/pharos/pki/etcd/client.pem --key /etc/pharos/pki/etcd/client-key.pem https://localhost:2379'
```

Comment:
`curl -sSf` would be best, in order to handle HTTP API errors as SSH exec errors.